src/transformers/jasmine-globals.test.ts

```diff
@@ -42,6 +43,7 @@ test('spyOn', () => {
     jest.spyOn(stuff).mockResolvedValue('lmao');
     jest.spyOn(stuff).mockRejectedValue('oh no');
     const fetchSpy = jest.spyOn(window, 'fetch').mockResolvedValue({json: {}});
+    existingSpy;
```

Comment:
This looks wrong to me.

Reply:
decided to make that change. let me know what you think.

Reply:
Seems to make sense, thanks.